### PR TITLE
To do list review

### PR DESCRIPTION
**Pull request by NalianAchei2020**
# 📝 Accomplishments
- Copy and paste code from the To_do-list project
- Update readME file

### CODE REVIEW

**Comment by George7h** [comment]
Hi @NalianAchei2020 ,

Good job on your project!! 👍 

Highlights 🟢 :

- Good use of JavaScript best practices.
- HTML & CSS best practices.

- According to DRY, KISS, and YAGNI rules, I have the following remarks:

- DRY:

The use of imported functions like 'addTask', 'removeTask', and 'editTaskt' promotes code reuse and avoids duplicating.

- KISS:

The code adheres to the KISS principle by maintaining a straightforward and understandable structure.

- YAGNI:

There are no indications of unnecessary features or over-engineering.



So basically all I can say is well done! 😄 😄 👍 



Your project is complete 👍 💯  There is nothing else to say other than... it's time to merge it 🕵️ 
Congratulations! 🎉
